RequestManagementSystem/Client/Request.py

```diff
@@ -370,7 +370,7 @@ def Status( self, value ):
     # If the status moved to Failed or Done, update the lastUpdate time
     if value in ( 'Done', 'Failed' ):
       if value != self._Status:
-        self.LastUpdate = datetime.datetime.utcnow().replace( microsecond = 0 )
```

Comment:
I saw that too, but you are using @hybrid_property, and not setter/getter/deleter or property.
Maybe I should make a pull request for pylint...